openapi.yaml

```diff
@@ -226,6 +226,52 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /help:
+    get:
+      summary: Retrieve help information for the given command.
+      operationId: help
+      tags:
+        - help
+      requestBody:
```

Comment:
@princerajpoot20 this endpoint receives the command in the URL path, not the body 🙂. Can you change this so that the path is `/help/{{comand}}`?

Reply:
Previously, you mentioned that there should be only one endpoint. You said, `for the API routes, try to only make one route that receives a parameter. For example you shouldn't have this get('/help/generate')`

Reply:
@princerajpoot20 in order to know if the URLs are possible, first you can think if this command from the CLI `generate modles fromTemplate ASYNCAPI TEMPLATE` is possible in our `server-api`? If it's possible in the API, it will be clearer how we should design the API